code/modules/projectiles/guns/beam_rifle.dm

```diff
@@ -47,13 +47,13 @@
 	var/wall_devastate = 0
 	var/aoe_structure_range = 1
 	var/aoe_structure_damage = 50
-	var/aoe_fire_range = 2
```

Comment:
this my friend is the rnd varient
what you want to do is make a new subtype
not make the rnd varient different